### ISSUE TITLE
discrepancy between GitHub tag v2.0.1 and npm installed version v2.0.1

### ISSUE THREAD

**Issue by jebeck** [jeffmo/jasmine-pit#8]
`jest-cli` installed `jasmine-pit` v2.0.1 for me, but I got the following error upon trying to run my tests with jest:

```
declare class jasmineIt {
        ^^^^^
Failed with unexpected error.
SyntaxError: Unexpected reserved word
```

Started investigating, and noticed that the index.js installed from npm via `jasmine-pit` v2.0.1 does not match the code under the v2.0.1 tag on GitHub [here](https://github.com/jeffmo/jasmine-pit/blob/v2.0.1/index.js).

Reverted to `jasmine-pit` v2.0.0 in `jest-cli`'s package.json and am able to run tests with jest.

**Comment by kkolstad-trov** [comment]
+1

**Comment by contolini** [comment]
:100: Jest is crashing hard.

**Comment by maspwr** [comment]
I just hit this as well.

**Comment by jeffmo** [comment]
Sorry about that! git-stash snafu... I just published 2.0.2

thanks for the report!
